src/app/happiness-tracker/happiness-tracker.component.html

```diff
@@ -1,15 +1,186 @@
-<div class="header">
-  <h1>
-    Happiness Tracker
-  </h1>
-  <div class="description">
-    Have you been wondering how your work is affecting your mood? How about your mental health?
-    Today is your lucky day! Use happy-dev's happiness tracker to see how your mood levels fluctuate day-to-day and task-to-task.
-  </div>
-</div>
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Montserrat:wght@500&family=Nunito+Sans:wght@200&display=swap" rel="stylesheet">
+</head>
 
-<div class="footer">
-  <p>Thanks for visiting happy-dev!</p>
-</div>
+<div class="happiness">
+  <h1>Pursue Happiness</h1>
+
+  <h2>Take our daily happiness quiz to determine how satisified you are as a developer</h2>
+
+  <form [formGroup]="form" (ngSubmit)="submit()">
+    <div class="form-group">
+      <div class="productivityQuestions">
+      <h3>Productivity</h3>
+      <label for="productivity1"><p>How often were you on your phone today?</p></label>
+        <input id="never" type="radio" value="never" name="productivity1" formControlName="productivity1">
```

Comment:
While many browsers don't fail when this happens, you cannot have duplicate id values in the same page. so when you're reusing "never" in others it's actually invalid HTML. I have some ideas on how you can make this more generated that perhaps we can talk about. I think it would be interesting for you to define the structure of the quiz you want in TS (or even a database). Then have the UI generate the form for you based on the structure of the data.